ecommerce_integrations/amazon/doctype/amazon_sp_api_settings/amazon_sp_api.py

```diff
@@ -0,0 +1,453 @@
+# Copyright (c) 2022, Frappe and contributors
+# For license information, please see license.txt
+
+
+import datetime
+import hashlib
+import hmac
+
+import boto3
```

Comment:
@ankush I think boto3 and requests(using in Test) would be fine?

Reply:
requests is not required IMO. It will never be removed from framework since it's super imp there. 

Also, boto requirements needs to loosen up a bit to avoid conflicts in future. I'll push the change after checking usage.

Reply:
Sorry my bad its "responses" not "requests"

Reply:
responses is already there in `dev-requirements.txt` so not required 😄